packages/playground/src/pages/Cards/Cards.js

```diff
@@ -5,20 +5,24 @@ import { handleSubmit, handleAdditionalDetails, handleError } from '../../handle
 import { amount, shopperLocale } from '../../config/commonConfig';
 import '../../../config/polyfills';
 import '../../style.scss';
+import { getSearchParameters } from '../../utils';
 
 getOriginKey()
     .then(originKey => {
         window.originKey = originKey;
     })
     .then(() => getPaymentMethods({ amount, shopperLocale }))
     .then(paymentMethodsResponse => {
+        //getPaymentMethods({ amount, shopperLocale }).then(paymentMethodsResponse => {
         window.checkout = new AdyenCheckout({
             amount, // Optional. Used to display the amount in the Pay Button.
-            originKey,
+            //            originKey,
             clientKey: process.env.__CLIENT_KEY__,
             paymentMethodsResponse,
             locale: shopperLocale,
+            ...(getSearchParameters().countryCode?.toLowerCase() === 'kr' && { countryCode: 'KR' }),
```

Comment:
this could be replaced by importing `countryCode` from `''../../config/commonConfig'`

Reply:
Being able to do this allows the test to run independently of how the playground is set up - meaning the test can always work without having to "reconfigure" the playground every time.
And it is only a playground, right?

Reply:
ok

Reply:
And do we want to always include `countryCode` in the config, which importing it from `commonConfig` will do?

Reply:
That should be all right 👍

Reply:
ok, made the change